src/lib/components/workflow-actions.svelte

```diff
@@ -100,17 +103,17 @@
         workflowId: workflow.id,
         runId: workflow.runId,
       });
+      cancelConfirmationModal.close();
```

Comment:
Should this be terminateConfirmationModal.close()?

Reply:
No, I think the diff is just misleading.

Reply:
ah yup you're right